### PR TITLE
use quickcache instead of custom cache

### PR DESCRIPTION
**Pull request by snopoke**
@millerdev

### CODE REVIEW

File: corehq/apps/receiverwrapper/util.py [hunk 1]
```diff
@@ -46,6 +45,7 @@ def get_meta_appversion_text(xform):
         return None
 
 
+@quickcache(['domain', 'build_id'], timeout=24*60*60)
```
Comment (dimagimon) [inline corehq/apps/receiverwrapper/util.py]: E226 missing whitespace around arithmetic operator

File: corehq/apps/receiverwrapper/util.py [hunk 2]
```diff
@@ -143,20 +133,11 @@ def get_build_version(xform):
     return None, BuildVersionSource.NONE
 
 
+@quickcache(['domain', 'build_or_app_id'], timeout=24*60*60)
```
Comment (dimagimon) [inline corehq/apps/receiverwrapper/util.py]: E226 missing whitespace around arithmetic operator

File: corehq/apps/receiverwrapper/util.py [hunk 3]
```diff
@@ -143,20 +133,11 @@ def get_build_version(xform):
...
-        if cache_value is None:
-            cache_value = _get_app_and_build_ids(domain, build_or_app_id)
-            cache.set(cache_key, cache_value, 24*60*60)
-        return cache_value
-    else:
-        app_id, build_id = build_or_app_id, None
-    return app_id, build_id
-
+    if not build_or_app_id:
+        return build_or_app_id, None
```
Comment (millerdev) [inline corehq/apps/receiverwrapper/util.py]: Is this always expected to return `None, None` or is `build_or_app_id` sometimes other falsy values like `''` or `0` etc?

Comment (snopoke) [inline corehq/apps/receiverwrapper/util.py]: Also wondered about that and wasn't sure so just left it as it was.

**Pull request merged**